fighters/falco/src/acmd/specials.rs

```diff
@@ -230,7 +210,7 @@ pub fn install() {
         falco_special_air_s_end_game,
         falco_special_hi_hold,
         falco_special_hi_game,
-        falco_special_lw_game,
+        // falco_special_lw_game,
```

Comment:
You can just delete this line rather than commenting out

Reply:
This was the recommendation from blujay but I dont care to delete it.